src/app/asset-page/artstor-viewer/artstor-viewer.component.ts

```diff
@@ -407,16 +407,19 @@ export class ArtstorViewerComponent implements OnInit, OnDestroy {
         });
 
         this.osdViewer.addOnceHandler("tile-drawn", () => {
-          // Get all thumbnails in the reference strip
-          let referenceStripThumbnails = document.querySelectorAll(".referencestrip .openseadragon-canvas");
-
-          referenceStripThumbnails.forEach((referenceStripThumbnail, index) => {
-            referenceStripThumbnail.addEventListener("keydown", (event: KeyboardEvent) => {
-             if (event.keyCode === 13) {
-               this.osdViewer.goToPage(index)
-             }
-           })
-          })
+            let mainContentCanvas: HTMLElement = <HTMLElement>document.getElementsByTagName("canvas")[0];
+            mainContentCanvas && mainContentCanvas.setAttribute("aria-label", this.assets[0].title ? this.assets[0].title : this.asset[0].description);
+
+            // Get all thumbnails in the reference strip
+            let referenceStripThumbnails = document.querySelectorAll(".referencestrip .openseadragon-canvas");
+
+            referenceStripThumbnails.forEach((referenceStripThumbnail, index) => {
+                    referenceStripThumbnail.addEventListener("keydown", (event: KeyboardEvent) => {
+                        if (event.keyCode === 13) {
```

Comment:
It still seems a little strange but I think if this line and the one above go back one indent it should be all aligned.

Reply:
Ahhh right!

Reply:
💯